pyfstat/make_sfts.py

```diff
@@ -129,10 +117,78 @@ def basic_setup(self):
             )
             for ind, dets in enumerate(IFOs)
         ]
+        self.IFOs = ",".join(['"{}"'.format(d) for d in IFOs])
+
+    def _get_setup_from_noiseSFTs(self):
+        noise_multi_sft_catalog = lalpulsar.GetMultiSFTCatalogView(
+            lalpulsar.SFTdataFind(self.noiseSFTs, self.SFTconstraint)
+        )
+
+        # Information to be extracted from the SFTs themselves
+        IFOs = []
+        tstart = []
+        tend = []
+        Tsft = []
+        self.sftfilenames = []  # This refers to the MFD output!
+
+        # SWIG-LAL does not import this functionality
+        gps_to_float = lambda x: x.gpsSeconds
```

Comment:
Shouldn't this be
```suggestion
        gps_to_int = lambda x: x.gpsSeconds
```
? In the no-noisesfts case, self.tstart and self.duration are also explicitly cast to int.